test-project/PlaybackViewController.m

```diff
@@ -39,11 +40,34 @@ - (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
   }
 }
 
+- (void) toggleFullscreen:(DMPlayerViewController *)player {
+
+    if (player.fullscreen) {
```

Comment:
Up in the `APIReadyWarn` method there's a big hunk of ternary operator where probably an if-else would read better, here though missed opportunity: `self.playerSize = player.fullscreen ? self.initialPlayerSize : self.view.frame.size`